src/main/java/uk/gov/hmcts/et/common/model/ccd/types/OrganisationAddress.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.et.common.model.ccd.types;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.jackson.Jacksonized;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Jacksonized
+@Builder
+public class OrganisationAddress {
+    @JsonProperty("addressLine1")
+    private String addressLine1;
+    @JsonProperty("addressLine2")
+    private String addressLine2;
+    @JsonProperty("addressLine3")
+    private String addressLine3;
+    @JsonProperty("townCity")
+    private String townCity;
+    @JsonProperty("county")
+    private String county;
+    @JsonProperty("country")
+    private String country;
+    @JsonProperty("postCode")
+    private String postCode;
+    // todo add dx array of something
```

Comment:
Is this still todo?

Reply:
No it isn't, I don't think we need it after all. I'll remove.